themes/scss/map/attributions.scss

```diff
@@ -40,6 +40,9 @@ $size: 20px;
   vertical-align: middle;
   z-index: 1;
 }
+.CDB-Attribution-button.disabled {
```

Comment:
@matallo I think in cartoassets we have a state for this one. I don't know if it's enough for this one.
![image](https://cloud.githubusercontent.com/assets/1214201/20667889/f45c436c-b56b-11e6-991c-bc9fa3720c48.png)